src/test/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServerSpec.scala

```diff
@@ -215,7 +214,7 @@ class JsonRpcHttpServerSpec extends AnyFlatSpec with Matchers with ScalatestRout
       status shouldEqual StatusCodes.TooManyRequests
     }
 
-    fakeClock.advanceTime(2 * serverConfigWithRateLimit.rateLimit.minRequestInterval.toMillis)
+    mockJsonRpcHttpServerWithRateLimit.mockedTime = 50000000L
```

Comment:
I think that you can use this example to pass a FakeTicker to advance the time at will.
https://github.com/google/guava/blob/master/guava-tests/test/com/google/common/cache/CacheRefreshTest.java